ui/components/app/signature-request-original/signature-request-original.container.js

```diff
@@ -202,25 +195,17 @@ function mergeProps(stateProps, dispatchProps, ownProps) {
   const mmiOnSignCallback = async (_msgData) => {
     if (accountType === 'custody') {
       try {
-        let msgData = _msgData;
-        let id = _msgData.custodyId;
-        if (!_msgData.custodyId) {
-          msgData = checkForUnapprovedMessages(
-            _msgData,
-            unapprovedPersonalMessages,
-          );
-          id = msgData.custodyId;
-        }
+        await dispatchProps.resolvePendingApproval(_msgData.id);
+        dispatchProps.completedTx(_msgData.id);
+
         dispatchProps.showCustodianDeepLink({
-          custodyId: id,
+          custodyId: null,
```

Comment:
Why do we need to pass null here?

Reply:
We don't have the custodyId at this point, so it goes as null.